include/blosc2.h

```diff
@@ -1804,6 +1804,34 @@ BLOSC_EXPORT int blosc2_schunk_get_chunk(blosc2_schunk *schunk, int64_t nchunk,
 BLOSC_EXPORT int blosc2_schunk_get_lazychunk(blosc2_schunk *schunk, int64_t nchunk, uint8_t **chunk,
                                              bool *needs_free);
 
+/**
+ * @brief Fill buffer with schunk slice.
```

Comment:
"Fill buffer with schunk slice." -> "Fill buffer with a schunk slice."

Reply:
Fixed in 1208e4e76f0f085baf1a4e02a8cc276654d40b0f.